### DISCUSSION TITLE
INTEL AMT CONNECT BUTTON

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6139] [Issues]
I just ran into an issue with the ‘Connect’ button not appearing under the Intel AMT tab.  I am perplexed since I activated it using amtconfig in the console of the NUC, and it shows that everything looks ok.  TLS is activated.  Not sure what it could be.  Would anyone be able to give me some ideas on this?
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/99822045-a702-4e0d-8f7a-3b694ee686c1)
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/ecd3672f-9f8c-4e74-ad30-1c7f834a129f)

I am getting the following error when I access mymeshcentral.com:4433.  What would be causing this?
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/30131fd2-2f24-44d4-a02e-55f60b0c82bd)

**Comment by mtran-88** [comment]
Hi, anyone has any suggestions or answer for this?

**Comment by silversword411** [comment]
Invalid cert authority. Use a valid one.
